src/unlimited.cpp

```diff
@@ -2133,7 +2133,7 @@ extern std::map<std::pair<void *, void *>, LockStack> lockorders;
 extern std::vector<std::string> vUseDNSSeeds;
 extern std::list<CNode *> vNodesDisconnected;
 extern std::set<CNetAddr> setservAddNodeAddresses;
-extern std::map<uint256, CTxCommitData> txCommitQ;
+extern std::map<uint256, CTxCommitData> *txCommitQ;
 extern std::queue<CTxInputData> txDeferQ;
```

Comment:
maybe use a unique pointer?  same as for the other pointer

Reply:
I'd prefer to control the deletion and have one less layer of abstraction we have to go through every time the pointer is dereferenced.